tests/src/rules/no-nodejs-modules.js

```diff
@@ -1,10 +1,13 @@
 import { test } from '../utils';
 
 import { RuleTester } from 'eslint';
+import semver from 'semver';
 
 const ruleTester = new RuleTester();
 const rule = require('rules/no-nodejs-modules');
 
+const supportsNodePrefix = semver.satisfies(process.versions.node, '^14.18 || >= 16', { includePrerelease: true });
```

Comment:
rather than using semver here, can we use `is-core-module` to see if each given module is supported?

Reply:
Thank you for your review quickly! I've fixed.